.mailmap

```diff
@@ -20,7 +20,7 @@ Jason McKerr <mckerrj@gmail.com> <jmckerr@jmckerr-OSX.local>
 Chris Houseknecht <chouseknecht@ansible.com>
 Chris Houseknecht <chouseknecht@ansible.com> chouseknecht <chouse@ansible.com>
 Chris Houseknecht <chouseknecht@ansible.com> Chris Houseknecht <chouse@ansibleworks.com>
-Stephen Fromm <sfromm@gmail.com> Stephen Fromm <stephenf@nero.net>
+Stephen From <sfromm@gmail.com> Stephen From <stephenf@nero.net>
```

Comment:
This is the name of the person. Please do not change it.

Reply:
race condition!!!